assets/js/plugin-check-admin.js

```diff
@@ -172,14 +172,41 @@
 	 * @param {Object} data The response data.
 	 */
 	async function runChecks( data ) {
+		let isSuccessMessage = true;
+		let messageClass = '';
+		let messageText = '';
 		for ( let i = 0; i < data.checks.length; i++ ) {
 			try {
 				const results = await runCheck( data.plugin, data.checks[ i ] );
+				const errorsLength = Object.values( results.errors ).length;
+				const warningsLength = Object.values( results.errors ).length;
```

Comment:
```suggestion
				const warningsLength = Object.values( results.warnings ).length;
```

Reply:
@mukeshpanchal27 This was dismissed but the change was not made. What happened here?